src/Confluent.Kafka/TopicPartitionOffsetError.cs

```diff
@@ -116,15 +125,52 @@ public override bool Equals(object obj)
             return tp.Partition == Partition && tp.Topic == Topic && tp.Offset == Offset && tp.Error == Error;
         }
 
+        /// <summary>
+        ///     Returns a hash code for this TopicPartitionOffsetError.
+        /// </summary>
+        /// <returns>
+        ///     An integer that specifies a hash value for this TopicPartitionOffsetError.
+        /// </returns>
         public override int GetHashCode()
-            => ((Partition.GetHashCode()*251 + Topic.GetHashCode())*251 + Offset.GetHashCode())*251 + Error.GetHashCode(); // x by prime number is quick and gives decent distribution.
+            // x by prime number is quick and gives decent distribution.
+            => ((Partition.GetHashCode()*251 + Topic.GetHashCode())*251 + Offset.GetHashCode())*251 + Error.GetHashCode();
```

Comment:
I don't think it matters much if the hash function is poor for this class, only that it is consistent with `Equals`. Anyway, I took the implementation from somewhere that claimed it was good (though it's possible I misinterpreted. I can no longer find the reference).